src/niscope/examples/niscope_fetch_forever.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import niscope
+import numpy as np
+import pprint
+import sys
+
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+# This example is based on the C FetchForever example
+def example(resource_name, channels, options, total_acquisition_time_in_seconds, voltage, sample_rate_in_hz, samples_per_fetch):
+    total_samples = int(total_acquisition_time_in_seconds * sample_rate_in_hz)
+    channel_list = channels.split(',')  # We need channels as a list
+
+    # 1. Creating numpy arrays
+    waveforms = [np.ndarray(total_samples, dtype=np.float64) for c in channel_list]
+
+    # 2. Opening session
+    with niscope.Session(resource_name=resource_name, options=options) as session:
+        # 3. Configuring
+        session.configure_horizontal_timing(min_sample_rate=sample_rate_in_hz, min_num_pts=1, ref_position=0.0, num_records=1, enforce_realtime=True)
+        session.channels[channel_list].configure_vertical(voltage, coupling=niscope.VerticalCoupling.DC, enabled=True)
+        # We configure for a software trigger and then never send it
```

Comment:
Can you explain why?

Reply:
Because that is how they did it in the C example. Other than that, no I cannot explain why.

Reply:
Does it still work if you stop doing it?

Reply:
I changed the comment to match what was in the C example. It looks like it is required.